### DISCUSSION TITLE
Lesson 6: Brownie Fund Me - CompilerError: File outside of allowed directories

### DISCUSSION THREAD

**Discussion by NibbledBit** [smartcontractkit/full-blockchain-solidity-course-py#600] [Q&A] [answered]
Hi there,

I've seen this error in discussion (and solved) so I've double checked and can't figure out why mine isn't working..
I'm on Windows 11.

Here's my brownie-config.yaml
![image](https://user-images.githubusercontent.com/82572031/146831666-fce33b52-acae-40d3-9d90-2c186541b3d9.png)

My sol which has an error not seen in the YouTube or the other similar problems:
![image](https://user-images.githubusercontent.com/82572031/146831770-c431575e-e0c5-4755-8b0c-fd12559a0ddd.png)

And the terminal error:
![image](https://user-images.githubusercontent.com/82572031/146831866-7eceb889-a151-431f-b772-e3b4bc0f0210.png)

Any help much appreciated, thanks so much!

**Comment by NibbledBit** [comment]
I've tried a different version:
![image](https://user-images.githubusercontent.com/82572031/146833761-0996e3fd-0511-4267-80ba-c39fb7bbcdad.png)

Not sure if this is successful
![image](https://user-images.githubusercontent.com/82572031/146834072-60d019cc-2ac7-451e-a746-80672200c89f.png)

**Comment by cromewar** [comment, marked as answer]
Hello @NibbledBit, don't forget to save the always the file.
To get rid off the linting issues please refer to [chronological-issues-from-video.md](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/blob/main/chronological-issues-from-video.md).
Regarding this compilation error, try deleting the build folder and try again, if this does not work please delete the `.brownie` folder on WSL home, and try again.

**Reply by NibbledBit** [reply]
Hi @cromewar, 
As you can see by the tab headers they are indeed saved... I'm guessing it's a build/cache issue too, I managed to finally deploy late last night in the end.
I can't find the .brownie folder is that on windows? how do I find it?

chronological-issues-from-video.md - brilliant! First I've seen this - thank you kindly!

![image](https://user-images.githubusercontent.com/82572031/146920444-f1dd65b3-43de-4050-aa57-3104c0fe7806.png)

**Reply by cromewar** [reply]
As your are using WSL the `.brownie` is on `~/` which is your `/home/yourUser/.brownie` deleting this folder helps on extreme cases, anyways did you solve the problem?

**Reply by NibbledBit** [reply]
Problem solved, thank you :)
